bindings/python/src/tokenizer.rs

```diff
@@ -270,7 +286,6 @@ impl FromPyObject<'_> for PyArrayUnicode {
                 (*arr).data,
             )
         };
-        let n_elem = array.shape()[0];
 
         // type_num == 19 => Unicode
         if type_num != 19 {
```

Comment:
I think this could be more readable by using the [`npyffi::types::NPY_TYPES::NPY_UNICODE`](https://docs.rs/numpy/latest/numpy/npyffi/types/enum.NPY_TYPES.html#variant.NPY_UNICODE) constant.